docs/Reference/Request.md

```diff
@@ -23,10 +23,13 @@ Request is a core Fastify object containing the following fields:
 - `host` - the host of the incoming request (derived from `X-Forwarded-Host`
   header when the [`trustProxy`](./Server.md#factory-trust-proxy) option is
   enabled). For HTTP/2 compatibility it returns `:authority` if no host header
-  exists. When you use `requireHostHeader = false` in the server options, it
-  will fallback as empty when the host header is missing.
-- `hostname` - the host of the incoming request without the port
-- `port` - the port that the server is listening on
+  exists. The host header may return empty string when using
+  `requireHostHeader = false`, not suppied when connected with `HTTP/1.0` or
+  using schema validation that remove the extra headers.
+- `hostname` - the host of the `host` property, it may refers the incoming
+  request hostname
+- `port` - the port of the `host` property, it may refers the port thats
```

Comment:
*refer

Reply:
@zdila this landed, could you send a fresh PR?